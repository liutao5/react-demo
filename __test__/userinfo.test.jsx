import React from 'react';
import renderer from 'react-test-renderer';
import Userinfo from '../src/pages/userinfo';

test('test Userinfo page', () => {
  const component = renderer.create(
    <Userinfo />,
  );
  const tree = component.toJSON();
  expect(tree).toMatchSnapshot();
});
