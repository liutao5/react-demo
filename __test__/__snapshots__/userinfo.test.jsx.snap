// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`test Userinfo page 1`] = `"userinfo"`;
